src/components/buttons/buttons.scss

```diff
@@ -1,12 +1,10 @@
 // Buttons
 //
-// <ul>
-//  <li>A `<button>` tag does not need anything special to work. Use `<button>` when you can, but it is possible to use other elements as long as you add `role="button"` and add JavaScript to replicate the button functionality.</li>
-//  <li>Just like links, you can add `class="visuallyhidden"` with descriptive text to give more context to the button's purpose.</li>
-//  <li>If a button contains an <code>&lt;img&gt;</code> element, make sure to set its `alt` attribute. If it contains an icon, use `aria-label` to describe the icon instead.</li>
-//  <li>You can use <code>&lt;input type=&quot;image&quot;&gt;</code> to make a graphical button. It takes a `src` and `alt` attribute just like traditional images.</li>
-//  <li>Button states are important, not just button styling! If you are only toggling classes to visually manage state of your components, you are likely not appropriately conveying that state to users of assistive technologies.</li>
-// </ul>
+//  - A `<button>` tag does not need anything special to work. Use `<button>` when you can, but it is possible to use other elements as long as you add `role="button"` and add JavaScript to replicate the button functionality.
```

Comment:
start off saying "tag" then call say "element" later.  We should probably take a look at the content in general and choose one word or another.